server/apps/publish/content/common.py

```diff
@@ -238,7 +238,11 @@ def _publish_takes_package(self, package, updates, original, last_updated):
                                              original_of_take_to_be_published=original,
                                              package=package)
 
-        self._set_updates(package, package_updates, last_updated)
+        # If the original package is corrected then the next take shouldn't change it back to 'published'
+        preserve_state = ITEM_STATE in package and package.get(ITEM_STATE, '') == CONTENT_STATE.CORRECTED and \
```

Comment:
this can be
`preserve_state = package.get(ITEM_STATE, '') == CONTENT_STATE.CORRECTED and updates.get(ITEM_OPERATION, ITEM_PUBLISH) == ITEM_PUBLISH`